locations/kranold18.yml

```diff
@@ -0,0 +1,100 @@
+---
+location: kranold18
+location_nice: Kranoldstr. 18
+latitude: 52.513189
+longitude: 13.464245
+community: true
+
+hosts:
+
+  - hostname: kranold18-core
+    role: corerouter
+    model: 'avm_fritzbox-4040'
+    wireless_profile: kranold18
+
+  - hostname: kranold18-ap1
+    role: ap
+    model: 'ubnt_nanostation_m5_xm'
+    wireless_profile: mesh_only
+
+ipv6_prefix: "2001:bf7:820:2800::/56"
+
+networks:
+  - vid: 20
+    role: mesh
+    name: mesh_philmel
+    prefix: 10.31.235.128/29
+    ipv6_subprefix: -3
+    mesh_metric: 2048
+    mesh_ap: kranold18-ap1
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+
+  - vid: 40
+    role: dhcp
+    inbound_filtering: true
+    enforce_client_isolation: true
+    prefix: 10.31.235.144/29
+    ipv6_subprefix: 0
+    assignments:
+      kranold18-core: 1
+
+  - vid: 41
+    role: dhcp
+    name: home
+    no_corerouter_dns_record: true
+    inbound_filtering: true
+    enforce_client_isolation: false
+    untagged: true
+    prefix: 10.31.235.152/29
+    ipv6_subprefix: 2
+    assignments:
+      kranold18-core: 1
+
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.235.136/29
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      kranold18-core: 1
+      kranold18-ap1: 1
+
+location__channel_assignments_11a_standard__to_merge:
+  kranold18-core: 48-20
+
+location__wireless_profiles__to_merge:
+  - name: kranold18
+    devices:
+      - radio: 11a_standard
+        legacy_rates: false
+        country: DE
+      - radio: 11g_standard
+        legacy_rates: false
+        country: DE
+
+    ifaces:
+      - mode: ap
+        ssid: kranold18.freifunk.net
+        encryption: none
+        network: dhcp
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: ff
+
+      - mode: ap
+        ssid: kranold18.freifunk.net Encrypted
+        hidden: true
+        encryption: owe
+        network: dhcp
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: ffowe
+        ieee80211w: 1
+
+      - mode: ap
+        ssid: kranold18-home.freifunk.net
```

Comment:
Since this is a private and password protected network I would leave "freifunk.net" out of there. Just name it "kranold18-home" or something similar.